docs/installation/ides.md

```diff
@@ -146,37 +146,19 @@ Emacs users have 3 options:
 
 ## PyCharm
 
-=== "PyCharm Community"
+PyCharm users can enable native basedpyright support in the settings:
 
-    1. install the [LSP4IJ](https://plugins.jetbrains.com/plugin/23257-lsp4ij) plugin
-    2. install the [Pyright](https://plugins.jetbrains.com/plugin/24145) plugin
-    3. configure it to use basedpyright by specifying `basedpyright-langserver` as the executable and check the "Resolve against interpreter directory, ignoring extension" checkbox:\
-        ![](./pycharm-lsp-exe.png)
-    4. set "Running mode" to "LSP4IJ":\
-        ![](./lsp4ij.png)
+1. Go to **Python | Tools | Pyright** in the Settings dialog.
+2. Select the **Enable** checkbox.
+3. In the Execution mode setting, select how PyCharm should search for the executable:
 
-=== "PyCharm Professional / IntelliJ IDEA Ultimate"
+    **Interpreter** mode: PyCharm searches for an executable installed in your interpreter. To install the basedpyright package for the selected interpreter, click _Install basedpyright_.
 
-    1. install the [Pyright](https://plugins.jetbrains.com/plugin/24145) plugin
-    3. configure it to use basedpyright by specifying `basedpyright-langserver` as the executable and check the "Resolve against interpreter directory, ignoring extension" checkbox:\
-        ![](./pycharm-lsp-exe.png)
-    3. set "Running mode" to "Native LSP client":\
-        ![](./native-lsp.png)
+    **Path** mode: PyCharm searches for an executable in `$PATH`. If the executable is not found, you can specify the path by clicking the Browse... icon.
 
-!!! tip "pinning basedpyright as a development dependency to your project (recommended)"
-
-    we recommend configuring these settings as overrides in the "Appearance & Behavior > Required Plugins" menu, and configuring the pyright plugin (and LSP4IJ if using pycharm community) as a recommended dependency:
-
-    ![](pycharm-recommended-dependency.png)
-
-
-    you should then commit the following generated config files, so that others working on your repo are prompted to install the plugin and don't have to manually configure it themselves:
-
-    - `.idea/pyright-overrides.xml`
-    - `.idea/pyright.xml`
-    - `.idea/externalDependencies.xml`
+4. Select which options should be enabled.
 
-    (note that pycharm hides the `.idea` directory by default, so you will need to `git add` the files via the CLI instead.)
+For more information, refer to [PyCharm documentation](https://www.jetbrains.com/help/pycharm/2025.3/lsp-tools.html#pyright).
```

Comment:
can we avoid hard-coding a version in the url?

Reply:
> otherwise the link might die if lsp-tools.html#pyright ever gets moved/renamed in a later version

i think it's better to point to latest/no version. sure the page might get renamed/moved, but that's better than referencing outdated docs. we can always update the link if it's moved, but needing to update it to point to a newer version seems annoying